main.go

```diff
@@ -276,19 +397,47 @@ func createMilestoneData(advance int, timeInterval string) map[string]milestone
 	return milestones
 }
 
-func createMilestones(baseURL string, token string, projectID string, milestones map[string]milestone) error {
+func createMilestones(baseURL string, token string, project string, milestones map[string]milestone, api string) error {
 	client := &http.Client{}
-	strURL := []string{baseURL, "/projects/", projectID, "/milestones"}
+	var strURL []string
+	switch {
+	case api == "gitlab":
+		strURL = []string{baseURL, "/projects/", project, "/milestones"}
+	case api == "github":
+		strURL = []string{baseURL, project, "/milestones"}
+	}
 	URL := strings.Join(strURL, "")
 	params := url.Values{}
 	for _, v := range milestones {
-		params.Set("title", v.Title)
-		params.Set("dueDate", v.DueDate)
-		req, err := http.NewRequest("POST", URL, strings.NewReader((params.Encode())))
-		if err != nil {
-			return err
+		var req *http.Request
+		switch {
+		case api == "gitlab":
+			params.Set("dueDate", v.DueDate)
+			params.Set("title", v.Title)
+			req, err := http.NewRequest("POST", URL, strings.NewReader((params.Encode())))
+			if err != nil {
+				return err
+			}
+			req.Header.Add("PRIVATE-TOKEN", token)
+		case api == "github":
+			create := struct {
```

Comment:
Don't completely like the inline struct here, but let's go with this for now.

Reply:
To me it felt like defining a struct at the top level just to send a few parameters to the API was even messier than just having it inlined

Reply:
Yeah true. Let's leave it for now. Make not be that messy after splitting it up into multiple files.